更新externalsType的三个变量的翻译 (#1555)
docschina/webpack.js.org: src/content/configuration/externals.mdx

### externalsType.module $#externalstype-module$

> 将 externals 类型设置为`'module'`， 它告诉 Webpack 在 module 中为外部引用生成形如 `import * as X from '...'`的代码。

Reviewer: ```suggestion
将 externals 类型设置为 `'module'`，webpack 将会在 module 中为外部引用生成形如 `import * as X from '...'` 的代码。
```

Reviewer: 中英文之间间隔一个空格哈

Reviewer: 这里的中英文好像不太对应，是参考的 webpack4 的文档吗

Author: 好的，externalsType应该是v5新特性，v4没找到，谢谢啦

> 确保首先把 [`experiments.outputModule`](/configuration/experiments/#experimentsoutputmodule) 设置为`true`, 否则 Webpack将会报错。

Reviewer: ```suggestion
确保首先把 [`experiments.outputModule`](/configuration/experiments/#experimentsoutputmodule) 设置为 `true`, 否则 webpack 将会报错。
```

> ### externalsType.node-commonjs

Reviewer: ```suggestion
### externalsType.node-commonjs $#externalstype-node-commonjs$
```

> 将 externals 类型设置为 `'node-commonjs'`， 它告诉 Webpack 在 module 中为外部引用引入`'module'`的 [`createRequire`](https://nodejs.org/api/module.html#module_module_createrequire_filename) 函数构造一个require函数

Reviewer: ```suggestion
将 externals 类型设置为 `'node-commonjs'`，webpack 将从 `module` 中导入 `createRequire` 来构造一个 require 函数，用于加载模块中使用的外部对象。
```

> ### externalsType.script

Reviewer: ```suggestion
### externalsType.script $#externalstype-script$
```

> 将 externals 类型设置为 `'script'`， 它告诉 Webpack 通过HTML中的 `<script>` 标签暴露预定义的全局变量的脚本，当脚本被加载完后 `<script>` 标签会被删除。

Reviewer: ```suggestion
将 externals 类型设置为 `'script'`，webpack 将会通过 HTML 中的 `<script>` 标签加载一个脚本，以暴露预定义的全局变量。当脚本被加载完后 `<script>` 标签会被删除。
```
